ffi/capi/src/fixed_decimal.rs

```diff
@@ -248,84 +263,10 @@ pub mod ffi {
-        #[diplomat::rust_link(
-            fixed_decimal::FixedDecimal::trunced_to_increment,
-            FnInStruct,
-            hidden
-        )]
-        pub fn trunc_to_increment(
-            &mut self,
-            position: i16,
-            increment: ICU4XFixedDecimalRoundingIncrement,
-        ) {
-            self.0.trunc_to_increment(position, increment.into())
-        }
-
-        #[diplomat::rust_link(fixed_decimal::FixedDecimal::half_trunc, FnInStruct)]
-        #[diplomat::rust_link(fixed_decimal::FixedDecimal::half_trunced, FnInStruct, hidden)]
-        pub fn half_trunc(&mut self, position: i16) {
-            self.0.half_trunc(position)
-        }
-
-        #[diplomat::rust_link(fixed_decimal::FixedDecimal::half_trunc_to_increment, FnInStruct)]
-        #[diplomat::rust_link(
-            fixed_decimal::FixedDecimal::half_trunced_to_increment,
-            FnInStruct,
-            hidden
-        )]
-        pub fn half_trunc_to_increment(
-            &mut self,
-            position: i16,
-            increment: ICU4XFixedDecimalRoundingIncrement,
-        ) {
-            self.0.half_trunc_to_increment(position, increment.into())
-        }
-
-        #[diplomat::rust_link(fixed_decimal::FixedDecimal::expand, FnInStruct)]
-        #[diplomat::rust_link(fixed_decimal::FixedDecimal::expanded, FnInStruct, hidden)]
-        pub fn expand(&mut self, position: i16) {
-            self.0.expand(position)
-        }
-
-        #[diplomat::rust_link(fixed_decimal::FixedDecimal::expand_to_increment, FnInStruct)]
-        #[diplomat::rust_link(
-            fixed_decimal::FixedDecimal::expanded_to_increment,
-            FnInStruct,
-            hidden
-        )]
-        pub fn expand_to_increment(
-            &mut self,
-            position: i16,
-            increment: ICU4XFixedDecimalRoundingIncrement,
-        ) {
-            self.0.expand_to_increment(position, increment.into())
-        }
-
-        #[diplomat::rust_link(fixed_decimal::FixedDecimal::half_expand, FnInStruct)]
-        #[diplomat::rust_link(fixed_decimal::FixedDecimal::half_expanded, FnInStruct, hidden)]
-        pub fn half_expand(&mut self, position: i16) {
-            self.0.half_expand(position)
-        }
-
-        #[diplomat::rust_link(fixed_decimal::FixedDecimal::half_expand_to_increment, FnInStruct)]
-        #[diplomat::rust_link(
-            fixed_decimal::FixedDecimal::half_expanded_to_increment,
-            FnInStruct,
-            hidden
-        )]
-        pub fn half_expand_to_increment(
-            &mut self,
-            position: i16,
-            increment: ICU4XFixedDecimalRoundingIncrement,
-        ) {
-            self.0.half_expand_to_increment(position, increment.into())
+        #[diplomat::rust_link(fixed_decimal::FixedDecimal::round, FnInStruct)]
+        #[diplomat::rust_link(fixed_decimal::FixedDecimal::rounded, FnInStruct, hidden)]
+        pub fn round(&mut self, position: i16) {
```

Comment:
nit: documentation on what round means here

Reply:
@eggrobin wdyt about this being the default `round` function? Alternatively we can call this `round_half_even` or similar.

Reply:
> wdyt about this being the default `round` function?

Sounds reasonable to me (so long as it is documented of course, which it now is).